test/parallel/test-abortsignal-drop-settled-signals.mjs

```diff
@@ -0,0 +1,130 @@
+// Flags: --expose_gc
+
+import * as common from '../common/index.mjs';
+
+if (common.isASan) {
+  common.skip('ASan messes with memory measurements');
+}
+
+import { describe, it } from 'node:test';
+
+const makeSubsequentCalls = (limit, done, holdReferences = false) => {
```

Comment:
```suggestion
function makeSubsequentCalls(limit, done, holdReferences = false) {
```

Nit... let's use the regular function syntax here. Not seeing much need for the `const ... = ...` syntax